roles/st2mistral/tasks/main.yml

```diff
@@ -31,8 +33,19 @@
   when:
     - st2mistral_version != "latest"
     - st2mistral_version != "present"
+  register: mistral_install_tagged
   tags: st2mistral
 
+- name: Enable mistral db upgrade
+  file:
```

Comment:
One final thing is missing: `become: yes` to make sure we delete the lock files for sure

Reply:
doh!